images/ansible-operator/base.Dockerfile

```diff
@@ -15,8 +16,9 @@ RUN mkdir -p /etc/ansible \
   && echo 'roles_path = /opt/ansible/roles' >> /etc/ansible/ansible.cfg \
   && echo 'library = /usr/share/ansible/openshift' >> /etc/ansible/ansible.cfg
 
-# Copy python dependencies specs to be installed using Pipenv
-COPY Pipfile* ./
+
+# Copy python dependencies (including ansible) to be installed using Pipenv
+COPY images/ansible-operator/${ANSIBLE_VERSION}/Pipfile* ./
```

Comment:
This isnt going to work anymore. TODO cp base.Dockerfile ../ansible-operator-2.11-preview/base.Dockerfile